simulator.go

```diff
@@ -266,14 +266,26 @@ func (h *simulatorAPIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 				http.Error(w, err.Error(), http.StatusInternalServerError)
 				return
 			}
-			logger = logger.New("id", container.ID[:8])
+			cLogger := logger.New("id", container.ID[:8])
```

Comment:
I thought we'd only want the client prefix in the log messages generated within runContainer(), but as you pointed out they're helpful for the termination logs I added, and probably also for other logs emitted further down, so I'll revert